utils/send_email.py

```diff
@@ -0,0 +1,13 @@
+from django.conf import settings
+from django.core.mail import send_mail
+
+
+def send_email_notification(subject, message, recipient_list):
+    email_from = settings.EMAIL_HOST_USER
+    send_mail(
+        subject,
+        message,
+        email_from,
+        recipient_list,
+        fail_silently=False,
```

Comment:
You are explicitly saying that you want it to raise an exception in case of failure, but you are not handling any exception!